docs/fr/user/installation.rst

```diff
@@ -35,25 +35,31 @@ Installation
 Sur un serveur dédié (méthode conseillée)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-wallabag utilise un grand nombre de bibliothèques pour fonctionner. Ces bibliothèques doivent être installées à l'aide d'un outil nommé Composer. Vous devez l'installer si ce n'est déjà fait et vous assurer que vous utilisez bien la version 1.2 (si vous avez déjà Composer, faite un ``composer selfupdate``).
+wallabag utilise un grand nombre de bibliothèques PHP pour fonctionner. Ces bibliothèques doivent être installées à l'aide d'un outil nommé Composer. Vous devez l'installer si ce n'est déjà fait et vous assurer que vous utilisez bien la version 1.2 (si vous avez déjà Composer, faite un ``composer selfupdate``).
 
 Installation de Composer :
 
 ::
 
     curl -s http://getcomposer.org/installer | php
 
-Vous pouvez trouver des instructions spécifiques `ici (en anglais) <https://getcomposer.org/doc/00-intro.md>`__ :
+Vous pouvez trouver des instructions spécifiques `ici (en anglais) <https://getcomposer.org/doc/00-intro.md>`__.
 
-Pour installer wallabag, vous devez exécuter ces deux commandes :
+Vous devez aussi installer node.js (via `la page de téléchargements <https://nodejs.org/en/download/>`__), npm (déjà installé avec node.js), bower et grunt:
```

Comment:
No need to install nodejs, npm, bower & grunt, composer installs nodejs-installer then npm installs the rest of it. https://github.com/wallabag/wallabag/blob/2.1/install.sh